dvol_python/test_dvol.py

```diff
@@ -350,7 +350,6 @@ def test_non_standard_branch(self, volume_name, branch_name,
         self.assertEqual(
             [['*', volume_name, branch_name]], [line.split() for line in rest])
 
-    @skip_if_go_version
     def test_log(self):
```

Comment:
I don't think this test is rigorous enough. For example, authorship or timestamps could break completely and this would not fail. Now would be a good time to improve this test (or break it into multiple smaller tests), however you don't need to if you don't want to.